go/cmd/vtctldclient/command/serving_graph.go

```diff
@@ -51,6 +51,14 @@ var (
 		Args:                  cobra.MinimumNArgs(1),
 		RunE:                  commandGetSrvKeyspaces,
 	}
+	// UpdateThrottlerConfig makes a UpdateThrottlerConfig gRPC call to a vtctld.
+	UpdateThrottlerConfig = &cobra.Command{
```

Comment:
I don't think this belongs with the `serving_graph.go` commands. maybe a new `../command/throttler.go` file?

Reply:
refactored to its own `throttler.go` file